scripts/jenkins/ardana/heat-ardana-deployerincloud-lite.yaml

```diff
@@ -187,31 +189,19 @@ outputs:
     description: Floating IP address of the deployer-controller node
     value: { get_attr: [deployer-controller-floatingip, floating_ip_address] }
 
-  deployer-net-ardana-ip:
-    description: IP address of the deployer-controller in the ardana network
-    value: { get_attr: [deployer-controller, networks, { get_resource: network_ardana }, 0]}
-
   deployer-net-mgmt-ip:
     description: IP address of the deployer-controller in the mgmt network
     value: { get_attr: [deployer-controller, networks, { get_resource: network_mgmt }, 0]}
 
-  # compute1
-  compute1-net-ardana-ip:
-    description: IP address of the compute1 node in the ardana network
-    value: { get_attr: [compute1, networks, { get_resource: network_ardana }, 0]}
-
-  compute1-net-mgmt-ip:
-    description: IP address of the compute1 node in the mgmt network
-    value: { get_attr: [compute1, networks, { get_resource: network_mgmt }, 0]}
-
-  # compute2
-  compute2-net-ardana-ip:
-    description: IP address of the compute2 node in the ardana network
-    value: { get_attr: [compute2, networks, { get_resource: network_ardana }, 0]}
+  # controller
+  controllers-net-mgmt-ips:
+    description: List of IPs of the controllers in the mgmt network
+    value: ""
```

Comment:
Can we add some comment here. That this is empty because we have no extra controller?